Sources/SourceKitLSPAPI/BuildDescription.swift

```diff
@@ -10,20 +10,18 @@
 //
 //===----------------------------------------------------------------------===//
 
-import struct Foundation.URL
-
-private import struct Basics.AbsolutePath
-private import func Basics.resolveSymlinks
-
-internal import SPMBuildCore
-
-// FIXME: should import these module with `private` or `internal` access control
-import class Build.BuildPlan
-import class Build.ClangModuleBuildDescription
-import class Build.SwiftModuleBuildDescription
-import struct PackageGraph.ResolvedModule
-import struct PackageGraph.ModulesGraph
-internal import class PackageModel.UserToolchain
+import Foundation
+import TSCBasic
+
+// Ideally wouldn't expose these (it defeats the purpose of this module), but we should replace this entire API with
+// a BSP server, so this is good enough for now (and LSP is using all these types internally anyway).
+import Basics
+import Build
+import PackageGraph
+import PackageLoading
+internal import PackageModel
+import SPMBuildCore
+import Workspace
```

Comment:
Nit: I think the convention is to only import what is used instead of whole module. Is that right, @MaxDesiatov ?

Reply:
I'm open to changing it specifically for TSC as a way of knowing which TSC APIs are being used. But there's really not point doing it for SwiftPM APIs.

Reply:
Sounds good!

Reply:
ideally, we would enabled "include what you use" (IWYU) against the project, and enforce it in CI.

https://github.com/swiftlang/swift/blob/main/docs/HowToGuides/RunningIncludeWhatYouUse.md